src/main/java/com/revature/beans/remote/Subtopic.java

```diff
@@ -4,9 +4,22 @@
 
 /**
  * Bean representing a subtopic persisted in the topic service.
+ * 
+ * <br>
+ * <br>
+ * <b>Last Modified:</b>
+ *  <pre style="margin:0;border:0;padding:0;">    15 April 2018</pre>
+ *  
+ * @see Topic
+ * 
+ * @author [see same bean in hydra-topic-service (1802-Matt)]  
```

Comment:
I rather have you lists out all the names in there than referencing another separate repository that they may or may not be able to find. Also, there is no @ author in that file, I will go bug Pooja for this.